azext_iot/_params.py

```diff
@@ -188,6 +189,14 @@ def load_arguments(self, _):
                          help='Child device list (comma separated) includes only non-edge devices.')
         context.argument('remove_all', options_list=['--remove-all', '-a'], help='To remove all children.')
 
+    with self.argument_context('iot hub distributed-tracing update') as context:
+        context.argument('sampling_mode', options_list=['--sampling-mode', '--sm'],
+                         help='Turns sampling for distributed tracing on and off. 1 is On and, 2 is Off.',
```

Comment:
The choice of using 1 and 2 instead of boolean is a bit strange. Do we have any reasoning here?

Reply:
According to the specs, the choices are On/Off only but the values saved in device twin is 1/2.